From 01b1c10adb3ec8ef2e3e00c1bca777b0b5305f3b Mon Sep 17 00:00:00 2001
From: Koen Kooi <koen@dominion.thruhere.net>
Date: Mon, 28 Apr 2014 22:01:03 +0200
Subject: [PATCH 1/4] MainWindow: set minimum size to 640x400

Signed-off-by: Koen Kooi <koen@dominion.thruhere.net>
---
 src/ui/MainWindow.ui | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/ui/MainWindow.ui b/src/ui/MainWindow.ui
index 523cbfd..cee01d4 100644
--- a/src/ui/MainWindow.ui
+++ b/src/ui/MainWindow.ui
@@ -12,8 +12,8 @@
   </property>
   <property name="minimumSize">
    <size>
-    <width>1024</width>
-    <height>600</height>
+    <width>640</width>
+    <height>400</height>
    </size>
   </property>
   <property name="baseSize">
-- 
1.9.2

