From f0bca64e11db3a51e9c5b5e61a59a05882def0b7 Mon Sep 17 00:00:00 2001
From: Koen Kooi <koen@dominion.thruhere.net>
Date: Tue, 29 Apr 2014 16:10:06 +0200
Subject: [PATCH 4/4] MainWindow: handle maptool widget destroy event

Signed-off-by: Koen Kooi <koen@dominion.thruhere.net>
---
 src/ui/MainWindow.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/ui/MainWindow.cc b/src/ui/MainWindow.cc
index aba2947..3e4ac34 100644
--- a/src/ui/MainWindow.cc
+++ b/src/ui/MainWindow.cc
@@ -617,7 +617,7 @@ void MainWindow::buildCommonWidgets()
     createDockWidget(engineeringView,new ParameterInterface(this),tr("Onboard Parameters"),"PARAMETER_INTERFACE_DOCKWIDGET",VIEW_ENGINEER,Qt::RightDockWidgetArea);
     createDockWidget(simView,new ParameterInterface(this),tr("Onboard Parameters"),"PARAMETER_INTERFACE_DOCKWIDGET",VIEW_SIMULATION,Qt::RightDockWidgetArea);
 
-    createDockWidget(pilotView,new QGCMapTool(this),tr("Map view"),"MAP_VIEW_DOCKWIDGET",VIEW_MISSION,Qt::RightDockWidgetArea);
+    menuActionHelper->createToolAction(tr("Map View"), "MAP_VIEW_DOCKWIDGET");
 
     menuActionHelper->createToolAction(tr("Status Details"), "UAS_STATUS_DETAILS_DOCKWIDGET");
 
-- 
1.9.2

